src/Bridge/Symfony/Bundle/DependencyInjection/Compiler/AnnotationFilterPass.php

```diff
@@ -13,15 +13,14 @@
 
 namespace ApiPlatform\Core\Bridge\Symfony\Bundle\DependencyInjection\Compiler;
 
-use ApiPlatform\Core\Bridge\Doctrine\MongoDbOdm\Filter\FilterInterface as MongoDbOdmFilterInterface;
+use ApiPlatform\Core\Exception\InvalidArgumentException;
```

Comment:
Should probably not throw our exception here, but something from https://github.com/symfony/symfony/tree/master/src/Symfony/Component/DependencyInjection/Exception